tasks/main.yml

```diff
@@ -58,6 +58,17 @@
       {{ arscc_path_to_service_binary }} {{ solr_service_name }} restart > /dev/null
   when: enable_solr_restart == true
 
+- name: Create limitnofile
+  template:
+    src: solr_limitnofile.j2
+    dest: /etc/security/limits.d
```

Comment:
A couple of issues: 
- Dest needs the full path to the file; otherwise the task would either fail, or result in a `j2` file being added to the server; i'm not sure which, but neither is preferable.
- Dest should include the service name also, since there will be multiple conf files if the server runs multiple solr versions.
  ```yaml
  dest: /etc/security/limits.d/{{ solr_service_name }}.limitnofile.conf
  ```

The rest of this looks spot on.

Reply:
@dale-c-anderson : Corrected.